ngraph/cmake/external_onnx.cmake

```diff
@@ -18,6 +18,7 @@ set(ONNX_GIT_REPO_URL https://github.com/onnx/onnx.git)
 set(ONNX_GIT_BRANCH rel-${ONNX_VERSION})
 set(NGRAPH_ONNX_NAMESPACE ngraph_onnx)
 set(ONNX_PATCH_FILE "${CMAKE_CURRENT_SOURCE_DIR}/cmake/patches/onnx_patch.diff")
+set(FETCHCONTENT_UPDATES_DISCONNECTED_EXT_ONNX ON CACHE BOOL "Update ext_onnx only during first cmake run" FORCE)
```

Comment:
This change will cause problems if ONNX library version will change.
After such update everyone who use incremental build will miss the change.

Reply:
can we switch to submodule as a solution?

Reply:
I'm not a big fan of git submodules but until ONNX create a new release this patch is needed and will cause rebuilds when configure step is run.

Current solution(master) is rebuilding ONNX library if someone rebuild OV witch reconfiguration: `cmake .. && make`.
There is no rebuilding of ONNX library if only `make` is called.

If it is a big deal we can move to submodule, but if we can live with it I would rather leave it as is.

Reply:
Another problem with current approach is CVS-57609, for some reason, fetch fails sporadically. 
I expect this will be fixed by sommodule.

Reply:
@ilya-lavrenov We've discussed it with @postrational and we are agreed to test the submodule approach. We are not sure yet, how to apply the patch in such a case. A solution can be onnx fork under openvinotoolkit organization (as a temporary solution - in the next onnx release expected changes should be applied).
Let's discuss the details offline.